nibabel/tests/test_volumeutils.py

```diff
@@ -700,7 +700,7 @@ def test_a2f_non_numeric():
     # Some versions of numpy can cast structured types to float, others not
     try:
         arr.astype(float)
-    except ValueError:
+    except (TypeError, ValueError):
```

Comment:
Why do you need this change in this PR?

Reply:
Ah, nope. Rebased onto the wrong branch.